src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/TransferIntoPrisonService.kt

```diff
@@ -30,21 +30,21 @@ import kotlin.Result.Companion.success
 
 @Transactional
 @Service
-class PrisonTransferService(
```

Comment:
This class is all about **inbound** transfers. Changed the name to fit with the existing `BookingIntoPrisonService`.